expo-maps/ios/ExpoMaps/AppleMapsView.swift

```diff
@@ -26,5 +26,44 @@ public final class AppleMapsView: UIView {
       mapViewType = .standard
     }
     self.mapView.mapType = mapViewType
+    
   }
+
+  func setCompassButton(enable: Bool) -> Void {
+    self.mapView.showsCompass = enable
+  }
+    
+  func setMyLocationButton(enable: Bool) -> Void {
+    if (enable == true) {
+      self.enableMyLocationButton()
+    }
+  }
+    
+  func setLevelPickerButton(enable: Bool) -> Void {
+    //TODO: enable floor picker
+  }
+    
+  private func enableMyLocationButton() -> Void {
+    self.requestLocationPermission()
+
+    self.mapView.showsUserLocation = true
+    let myLocationButton = MKUserTrackingButton(mapView: self.mapView)
+        myLocationButton.layer.backgroundColor = UIColor(white: 1, alpha: 0.5).cgColor
+        myLocationButton.layer.borderColor = UIColor.white.cgColor
+        myLocationButton.layer.borderWidth = 1
+        myLocationButton.layer.cornerRadius = 5
+        myLocationButton.translatesAutoresizingMaskIntoConstraints = false
+    self.mapView.addSubview(myLocationButton)
+          
+    NSLayoutConstraint.activate([
+        myLocationButton.topAnchor.constraint(equalTo: self.mapView.topAnchor,
+            constant: 100),
+        myLocationButton.trailingAnchor.constraint(equalTo: self.mapView.trailingAnchor, constant: -10)])
+  }
```

Comment:
```suggestion
    NSLayoutConstraint.activate([
        myLocationButton.topAnchor.constraint(equalTo: self.mapView.topAnchor, constant: 100),
        myLocationButton.trailingAnchor.constraint(equalTo: self.mapView.trailingAnchor, constant: -10)
    ])
  }
```
It looks better in my opinion ;)